src/settings/LostItemFeePolicy/components/EditSections/LostItemFeeSection/LostItemFeeSection.test.js

```diff
@@ -0,0 +1,474 @@
+          name: 'replacedLostItemProcessingFee',
+          id: 'replacedLostItemProcessingFee',
+          component: Select,
+          dataOptions: expectedDataOptions,
+        }, {}
+      );
+    });
+
+    it('should render "replacement process fee" field', () => {
+      expect(Field).toHaveBeenNthCalledWith(
+        fieldCallOrderByPlace.replacementProcessFee,
+        expect.objectContaining({
+          'aria-label': labelIds.replacementProcessFee,
+          label: labelIds.replacementProcessFee,
+          name: 'replacementProcessingFee',
+          id: 'replacement-processing-fee',
+          component: TextField,
+          type: 'number',
+          formatOnBlur: true,
+        }), {}
+      );
+    });
+
+    it('should render "charge amount item patron" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.chargeAmountItemPatron,
+        expect.objectContaining({
+          htmlFor: 'charge-amount-item-patron',
+          id: 'charge-amount-item-patron-label',
+        }), {}
+      );
+    });
+
+    it('should render "charge amount item system" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.chargeAmountItemSystem,
+        expect.objectContaining({
+          htmlFor: 'charge-amount-item-system',
+          id: 'charge-amount-item-system-label',
+        }), {}
+      );
+    });
+
+    it('should render "returned lost item processing fee" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.returnedLostItemProcessingFee,
+        expect.objectContaining({
+          htmlFor: 'returnedLostItemProcessingFee',
+          id: 'returnedLostItemProcessingFee-label',
+        }), {}
+      );
+    });
+
+    it('should render "lost item returned" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.lostItemReturned,
+        expect.objectContaining({
+          htmlFor: 'Charge',
+          id: 'Charge-label',
+        }), {}
+      );
+    });
+
+    it('should render "replacement allowed" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.replacementAllowed,
+        expect.objectContaining({
+          htmlFor: 'replacementAllowed',
+          id: 'replacementAllowed-label',
+        }), {}
+      );
+    });
+
+    it('should render "replaced lost item processing fee" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.replacedLostItemProcessingFee,
+        expect.objectContaining({
+          htmlFor: 'replacedLostItemProcessingFee',
+          id: 'replacedLostItemProcessingFee-label',
+        }), {}
+      );
+    });
+
+    it('should render radio group', () => {
+      expect(RadioGroup).toHaveBeenCalled();
+    });
+
+    it('should render "chargeAmountItemPatron" text', () => {
+      expect(
+        within(screen.getByTestId('chargeAmountItemPatronColumn'))
```

Comment:
we can use `const getById = (id) => within(screen.getByTestId(id))`, and then just getById(id).getByText(text). and as you mentioned earlie, when we will extract `getById` in more global level, we can just remove function definition and add import without any renaming. and i think we should do extracting asap.
CC: @Dmitriy-Litvinenko

Reply:
agree. done